compiler/optimizer/LoopReducer.cpp

```diff
@@ -1750,14 +1741,7 @@ TR_Arraytranslate::checkStore(TR::Node * storeNode)
       }
    else
       {
-      if (storeNode->getOpCodeValue() == TR::cstorei)
```

Comment:
`== TR::sstorei` I think is more appropriate

Reply:
The mistake was fixed in bd43e4f